addons/mail/data/mail_channel_demo.xml

```diff
@@ -163,5 +163,7 @@
             <field name="date" eval="(DateTime.today() - timedelta(days=3)).strftime('%Y-%m-%d %H:%M')"/>
         </record>
 
+        <function model="mail.channel" name="_mark_seen_general_channel"/>
```

Comment:
According to what I just posted above, I didn't pay attention but this call should actually be in real data, not demo data.